adobe/spectrum-css
# feat(storybook)!: update chromatic add-on to latest version

## Description

Chromatic moved away from `@chromaui/addon-visual-tests` to the `@chromatic-com/storybook` package.

This requires an update to the `chromatic.config.json` settings, removal of the generic argTypesRegex from the preview config, and a stricter import in the doc blocks (i.e., `fileName.js` not just `fileName`).

To support a successful build, our test command must maintain the autodocs and mdx syntax which requires we add this information to our build instructions in the .storybook/main.js file.

## How and where has this been tested?

Please tag yourself on the tests you've marked complete to confirm the tests have been run by someone other than the author.

### Validation steps

Expect to be able to run Storybook locally, connect to the Chromatic add-on in the component panel, and kick off a manual re-run of VRTs.

## Screenshots



## To-do list
